test/xml-generator/Class.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<SystemClass Name="Test">
+    <SubsystemInclude Path="Subsystem.xml"/>
+</SystemClass>
```

Comment:
Well... this is the SystemClass. If it's not defined, the Parameter Framework will not start.